RULES.md

```diff
@@ -0,0 +1,150 @@
+# Based Rules
+
+## Rationale
+
+These rules are implemented to ensure:
+- [based.cooking](https://based.cooking) stays free (libre; as in freedom),
+- the repository stays neat and tidy,
+- a consistent Markdown style amongst all recipes,
+- consistent technical details amongst all recipes,
+- appropriate tags and images representative of the recipes on which they are included,
+- no bloat or soydev code is permitted,
+- recipes are not condescending or verbose, and
+- only authentic, traditional, high-quality food is permitted.
+
+Make [based.cooking](https://based.cooking) based again!
+
+## Rules
+
+### Freedom
+
+#### License
+
+When you submit anything to [based.cooking](https://based.cooking), you agree to release your work under the CC0 1.0 license.
+Everything in the [based.cooking](https://based.cooking) repository is licensed under the CC0 1.0 license.
+Contributions licensed under any incompatible license will be rejected.
+
+#### Requests for removal
+
+Requests to have your work removed will be ignored.
+
+#### Attribution
+
+Attribution is unnecessary under the CC0 1.0 license.
+However, you are welcome and recommended to give yourself credit at the bottom of a recipe you submitted, including links to your personal website and to your donation pages.
+**Do not** link to social media sites.
+
+### Recipe format
+
+Model submission files after [example.md](example.md), then put them in `src/`.
+
+### Markdown style
+
+Recipes should start with a title, with a single `#` *on the first line*.
+
+### Technical details
+
+#### File format
+
+- Recipes should start with no empty line at the top and end with no trailing lines.
+- There should be no trailing whitespace.
+- The file needs to be `\n` terminated in Unix-like fashion (if you're using a Unix-like system, it should be automatic).
+- Encode files in UTF-8.
+
+#### File name
+
+- File names should be the name of the dish with words separated by hyphens (`-`), not underscores, and definitely not spaces.
+
+#### Tags
+
+- Recipes should end with a list of relevant tags.
+The syntax is:
+
+```md
+;tags: tag1 tag2 tag3
+```
+
+- The tag line should be a single line, at the end of the markdown file, preceded by a blank line.
+- Add between a few (1-4) tags.
```

Comment:
The CI script currently makes you have 2 to 5. I personally don't really care what the value is (I made up the 2 to 5 when I wrote it) but just making a note to make sure we make changes to make this rule consistent.

Reply:
1-4 is from Luke, although I would personally say 1-6 is the right ballpark.

Reply:
1-6 sounds good, 7 seems like too many